src/config.ts

```diff
@@ -25,6 +25,10 @@ const config = {
         PROD: 'https://op-build-prod.azurewebsites.net',
         PPE: 'https://op-build-sandbox2.azurewebsites.net'
     },
+    AIKey: {
+        PROD: '4424c909-fdd9-4229-aecb-ad2a52b039e6',

```

Comment:
where are these AI instances created?

Reply:
PROD: Open Publishing - Build Production
PPE: Open Publishing - Build SandBox


---
In reply to: [377450382](https://github.com/docascode/vscode-docs-build/pull/35#discussion_r377450382) [](ancestors = 377450382)

Reply:
what is the difference between PROD and PPE?

---
In reply to: [377464736](https://github.com/docascode/vscode-docs-build/pull/35#discussion_r377464736) [](ancestors = 377464736,377450382)

Reply:
Since extension support two Environment: 'PROD' and 'PPE', so I create two AI instance, PPE is for dev usage, so it won't affect the PROD data

Reply:
Since extension support two Environment: 'PROD' and 'PPE', so I create two AI instances, PPE is for dev usage, so it won't affect the PROD data

---
In reply to: [377488798](https://github.com/docascode/vscode-docs-build/pull/35#discussion_r377488798) [](ancestors = 377488798,377464736,377450382)